packages/editor/src/components/parts/program/editor/Editor.css

```diff
@@ -1,6 +1,9 @@
-.message {
+.infoTextArea {

```

Comment:
currently classes are all written with '-' -> so `info-text-area`
But as this is no infotext i would rename it to something like `info-text`

Reply:
```css
.info-text {
  display: flex;
  gap: ...
  padding-inline: ...
  font-size: 12px;
}
.info-text p {
  margin: 0;
}```